volumina/navigationController.py

```diff
@@ -148,7 +148,10 @@ def onWheel_default(self, imageview, event):
         grviewCenter = imageview.mapToScene(imageview.viewport().rect().center())
 
         sign = 1
-        if event.angleDelta().y() < 0:
+        if (k_ctrl or k_shift) and event.angleDelta().y() < 0:
+            sign = -1
+        elif (k_alt or k_shift_alt) and event.angleDelta().x() < 0:
+            # Qt.AltModifier maps vertical (y) to horizontal (x) scrolling direction
```

Comment:
```suggestion
        # Qt.AltModifier maps vertical (y) to horizontal (x) scrolling direction
        if sys.platform != "darwin" and (k_alt or k_shift_alt) and event.angleDelta().x() < 0:
            sign = -1
        elif event.angleDelta().y() < 0:
            sign = -1
```

the link to the bugtracker was helpful, as for me on a mac, the patch did the opposite: from working on the main branch to broken with the PR :). So I'd suggest to do something like the above. One could also use `platform.system()` to be more modern I guess, but `sys.platform` should suffice. (Note: code wasn't verified on Windows or Linux).